app/services/foreman_templates/action.rb

```diff
@@ -33,7 +33,9 @@ def get_absolute_repo_path
     end
 
     def verify_path!(path)
-      raise "Using file-based synchronization, but couldn't find #{path}" unless Dir.exist?(path)
+      msg = _("Using file-based synchronization, but couldn't access %s to export templates. ") % path
+      msg += _("Please check the access permissions/SELinux and make sure it is writable for 'foreman' system user.")
```

Comment:
Dev setups might run under different user. Can we either figure out current effective user or rephrase to something like `the web application user account, typically 'foreman'`